tensorboard/webapp/metrics/views/main_view/pinned_view_component.ts

```diff
@@ -51,5 +51,5 @@ import {CardIdWithMetadata} from '../metrics_view_types';
 export class PinnedViewComponent {
   @Input() cardObserver!: CardObserver;
   @Input() cardIdsWithMetadata!: CardIdWithMetadata[];
-  @Input() newCardPinnedIds!: [number];
+  @Input() lastPinnedCardTime!: [number];
```

Comment:
Does this still need to be a `[number]`? It seems simpler to be `number`

Reply:
Done.